style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  background-color: rgb(141, 107, 56);
+}
+
+.classbody {
+  background-color: rgb(211, 162, 82);
+  width: 432px;
+  margin-left: 500px;
+}
+
+h2 {
+  font-family: "Anton", sans-serif;
+  margin-left: 50px;
+}
+
+p {
+  font-family: "Noto Sans JP", sans-serif;
+  text-transform: uppercase;
+  margin-left: 50px;
+}
+
+.small-logo {
+  float: right;
```

Comment:
Because you have used `float`, you must also clear your floats. See my docs on [using `float`](https://mica-web.github.io/learn/css/float) for more info on this.